### DISCUSSION TITLE
Tampermonkey

### DISCUSSION THREAD

**Discussion by Cdrwbw** [fedorovvl/tso_client#74] [General]
It seems that copying the content of content.zip to use the client launched by the script (tampermonkey or other) does not work.
Any ideas? A solution?

**Comment by fedorovvl** [comment]
since 1.5.1 there is no content.zip file in repo..
you need download release source code archive ( Source code (zip) in assets) and extract content folder (client/files/content)
i put file The Settlers Online.exe in it for compatibility.. so just replace all files without renaming

**Reply by Cdrwbw** [reply]
Well, It's ok, and ubicollect too with client launch by script in browser
